core/src/main/java/hudson/slaves/ChannelPinger.java

```diff
@@ -42,7 +42,7 @@
 import java.util.logging.Logger;
 
 /**
- * Establish a periodic ping to keep connections between {@link Slave slaves}
+ * Establish a periodic ping to keep connections between {@link Slave agents}
  * and the main Jenkins node alive. This prevents network proxies from
```

Comment:
They're not as important but some other usages of "slave" around line 91 could easily be changed.